RELEASES.md

```diff
@@ -28,12 +28,19 @@ Decide among the maintainers if a new release is due.
 Dear all,
 
 Today, $NEW_VERSION of the openstreetmap-carto stylesheet (the default
-stylesheet on openstreetmap.org) has been released.
-
-If the tile server admins have rolled out the new version, add
-
-  and rolled out to the openstreetmap.org servers. It might take a
-  couple of days before all tiles show the new rendering.
+stylesheet on openstreetmap.org) has been released. The actual deployment 
+on OSM website is done by sysadmin team, so there are no guarantees 
+if and when the changes will be visible there.
```

Comment:
I don't like the tone of this sentence, it seems to be passively-aggressively blaming them that it's not rolled out.

Reply:
Another wording: `so please be patient, they make their own decisions if and when should it happen.`

Reply:
I agree that tone is wrong.

Maybe something like "Once changes are deployed on the openstreetmap.org it will take couple of days before all tiles show the new rendering"

Reply:
@pnorman What you think about new wording? I think that it is now OK.